test/integration/local/test_serving.py

```diff
@@ -95,7 +95,7 @@ def _assert_prediction_csv(test_loader, accept):
 
 def _get_test_data_loader(batch_size):
     return torch.utils.data.DataLoader(
-        datasets.MNIST(training_dir, train=False, transform=transforms.Compose([
+        datasets.MNIST(training_dir, train=False, download=True, transform=transforms.Compose([
```

Comment:
it should be using the dataset python sdk uploads to s3 and then sagemaker downloads to the channel directory on the box. 
It shouldn't be downloading the dataset from the internet every time.